test/unit/alignment/pairwise/execution/alignment_executor_two_way_test.cpp

```diff
@@ -10,8 +10,8 @@
 #include <string>
 
 #include <seqan3/alignment/pairwise/execution/alignment_executor_two_way.hpp>
-#include <seqan3/range/view/persist.hpp>
-#include <seqan3/range/view/view_all.hpp>
+#include <seqan3/range/views/persist.hpp>
+#include <seqan3/range/views/view_all.hpp>
```

Comment:
❌?

Reply:
Why did you include all of these in the first place :see_no_evil:

Reply:
All view is needed btw, because you use all_view ...